crates/rpc/rpc-types/src/eth/engine/payload.rs

```diff
@@ -111,6 +113,8 @@ impl From<SealedBlock> for ExecutionPayload {
             timestamp: value.timestamp.into(),
             extra_data: value.extra_data.clone(),
             base_fee_per_gas: U256::from(value.base_fee_per_gas.unwrap_or_default()),
+            blob_gas_used: Some(U64::from(value.blob_gas_used.unwrap_or_default())),
+            excess_blob_gas: Some(U64::from(value.excess_blob_gas.unwrap_or_default())),
```

Comment:
Now that both values are `Option`s, we should not be using `unwrap_or_default`, because it would put `Some(U64::default())` elements when the original value could be directly set as `None`
```suggestion
            blob_gas_used: value.blob_gas_used.map(U64::from),
            excess_blob_gas: value.excess_blob_gas.map(U64::from),
```